commit d3d3216a38349db449126daad41bc22324e98190 (refs/changes/00/3800/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-04-10T16:30:43-07:00 (1 year, 6 months ago)
    
    TRITON-304 node-triton test crash in 'triton inst nic ...' tests (extra output in tests to help diagnose failure)

diff --git a/test/integration/cli-nics.test.js b/test/integration/cli-nics.test.js
index 7850673..f4fe61b 100644
--- a/test/integration/cli-nics.test.js
+++ b/test/integration/cli-nics.test.js
@@ -52,6 +52,7 @@ test('triton instance nics', OPTS, function (tt) {
             if (h.ifErr(t, err, 'triton instance create'))
                 return t.end();
 
+            t.ok(instId, 'created instance ' + instId);
             INST = instId;
 
             t.end();
@@ -78,6 +79,7 @@ test('triton instance nics', OPTS, function (tt) {
                 return t.end();
 
             NIC = JSON.parse(stdout);
+            t.ok(NIC, 'created NIC: ' + stdout.trim());
 
             t.end();
         });
